stage0/run.go

```diff
@@ -406,6 +430,10 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 
 	destRootfs := common.Stage1RootfsPath(dir)
 
+	if len(cfg.DNS) > 0 || len(cfg.DNSSearch) > 0 || len(cfg.DNSOpt) > 0 {
```

Comment:
libc does not recognize more than 3 DNS servers. We should error out if more than 3 are supplied.